### DISCUSSION TITLE
Hashicorp vault backup using Velero

### DISCUSSION THREAD

**Discussion by rizwank23** [vmware-tanzu/velero#3850] [Community support Q&A]
We have vault deployed in our dev environment Digitalocean kubernetes cluster. So we want to take the backup of vault using velero and restore the the same in different cluster.

We are facing the issue like it is not getting restored properly and one more important thing is, the vault should be unsealed using the same old keys which we got while deploying in dev cluster. But this is not happening. It says vault should be initialised.

Any help would be apperciated

**Comment by chrislinan** [comment]
Are you using etcd-operator in vault ?
https://github.com/coreos/etcd-operator
